api/envoy/api/v2/route/route_components.proto

```diff
@@ -353,6 +353,9 @@ message RouteMatch {
   message TlsContextMatchOptions {
     // If specified, the route will match against whether or not a certificate is presented.
     google.protobuf.BoolValue presented = 1;
+
+    // If specified, the route will match against whether or not a certificate is validated.
```

Comment:
What is the default here? Can you clarify?

Reply:
If not specified, then the validated status (true or false) will not be considered for route matching - it's the same behaviour as the 'presented' field above.

Reply:
OK can you do me a favor and clarify the docs both here and above that no value means not considered at all, a set value checks that state, etc. Feel free to phrase however you want but it was a bit unclear to me from a quick doc read. Thank you!

Reply:
I've updated it, let me know if it's ok. Thanks.